.github/workflows/test-integration-dontrun.yml

```diff
@@ -0,0 +1,20 @@
+name: Integration Tests do not run on PRs
```

Comment:
This file is not working as intended. The required check has a different name according to github. Thus it doesn't show up in automerge and breaks the merge queue (which needs the required checks). Let's remove it and instead we can skip steps in the actual integration tests work if the PR is from a fork. The syntax 

```
if: github.event.pull_request.head.repo.full_name == github.repository  # then NOT on fork
```

from this [forum question](https://github.com/orgs/community/discussions/25217) should do the trick.

Reply:
You'll want this syntax actually

```
if: success() && github.event.pull_request.head.repo.full_name == github.repository
```

Reply:
@ytausch see my note here for another approach that allows PRs from branches to run the integration tests.

Reply:
I think I don't follow completely. Why does the required check have a different name? I think the status check should always be named after the ID (YAML key) of the job.

I have it from here: https://github.com/orgs/community/discussions/51120#discussioncomment-7589658

I see that your solution also works, but I want to avoid copy-pasting this if-condition 20 times if it's avoidable.

Reply:
Here, I tried out the approach that I've implemented so far: https://github.com/ytausch-terraform-tests/test-env/pull/11

Feel free to check out the CI logs to verify that the right workflows are executed. You are also welcome to submit a PR from your fork to try it out yourself!

Reply:
Ahh, I think you should not make `setup-repositories` a required status check for now, only `run-test-scenario`.

Reply:
It now works!